packages/components/src/Notification/_docs/GlobalNotification.mdx

```diff
@@ -0,0 +1,34 @@
+import { Canvas, Controls, Meta, Description} from "@storybook/blocks"
+import { Installation, KaioNotification, ResourceLinks } from "~storybook/components"
+import * as GlobalNotificationExamples from "./GlobalNotification.stories"
+
+<Meta title="Components/Notifications/Global Notification" />
+
+# GlobalNotification
```

Comment:
I'd prefer we have each of the component folders house their own `_docs`

Reply:
Good point, makes sense for house keeping to group them into to their own folders, especially when we consider that Toast will have to be added to the mix